apps/meteor/server/lib/systemMessage/hideSystemMessage.ts

```diff
@@ -0,0 +1,21 @@
+import type { MessageTypesValues, SettingValue } from '@rocket.chat/core-typings';
+import { Settings } from '@rocket.chat/models';
+import mem from 'mem';
+
+export const isMutedUnmuted = (messageType: string): boolean => {
+	return messageType === 'user-muted' || messageType === 'user-unmuted';
+};
+
+const getSettingCached = mem(async (setting: string): Promise<SettingValue> => Settings.getValueById(setting), { maxAge: 10000 });
+
+export const getCachedHiddenSystemMessage = async () => {
+	return getSettingCached('Hide_System_Messages') as unknown as MessageTypesValues[];
+};
```

Comment:
Do you use these for something? on `lib/messages` L25 you could use these but you're re-creating a mem there.

If no use for these, remove